src/test/java/uk/gov/pay/adminusers/utils/date/DisputeEvidenceDueByDateUtilTest.java

```diff
@@ -23,11 +23,11 @@ void shouldCorrectlyCalculatePayDueByDate(ZonedDateTime dateTime, DayOfWeek expe
 
     private static Stream<Arguments> epochProvider() {
         return Stream.of(
-                Arguments.of(parse("2022-03-07T13:00:00.000Z"), DayOfWeek.FRIDAY),
-                Arguments.of(parse("2022-03-08T13:00:00.000Z"), DayOfWeek.FRIDAY),
-                Arguments.of(parse("2022-03-09T13:00:00.000Z"), DayOfWeek.MONDAY),
-                Arguments.of(parse("2022-03-10T13:00:00.000Z"), DayOfWeek.TUESDAY),
-                Arguments.of(parse("2022-03-11T13:00:00.000Z"), DayOfWeek.WEDNESDAY)
+                Arguments.of(parse("2022-03-07T13:00:00.000Z"), DayOfWeek.MONDAY),
+                Arguments.of(parse("2022-03-08T13:00:00.000Z"), DayOfWeek.TUESDAY),
+                Arguments.of(parse("2022-03-09T13:00:00.000Z"), DayOfWeek.WEDNESDAY),
+                Arguments.of(parse("2022-03-10T13:00:00.000Z"), DayOfWeek.THURSDAY),
+                Arguments.of(parse("2022-03-11T13:00:00.000Z"), DayOfWeek.FRIDAY)
```

Comment:
It looks like all these different parameters were here to test special cases we had for making the due date is not on a weekend. Since we just always have a week before now, we probably don’t need all of these now. We could just have a single test that inputs a Stripe due date and checks the Pay due date is a week before (maybe make the Stripe due to the 3 January or something so the Pay due date is in a previous month and year to make it interesting!).

Reply:
Good spot! Will update it.